packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -391,6 +391,7 @@ export interface CallCompositeStrings {
     complianceBannerTranscriptionStopped: string;
     configurationPageCallDetails?: string;
     configurationPageTitle: string;
+    copyInviteLinkButtonLabel: string;
```

Comment:
Should this be in stable?
I am assuming this is new feature specific update

Reply:
Good point! As per the other review comment below, I have made changes to remove any api related code out. 
Those will come in a later PR which will target beta :)! Thanks Anjul!